### DISCUSSION TITLE
Renovate doesn't identify a second regex update

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#27829] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Same package rule is used to match two files, each with an identical regex annotation.  One update is identified while the second one not.

Reproduction repo - https://github.com/platform-engineering-org-test/renovate_test_2

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment]
please update repo readme to include current and expected behavior. you didn't say which one work and which not.

**Reply by lmilbaum** [reply]
Updated. Thanks for your feedback.

**Reply by viceice** [reply]
the second regex seems to miss `\\s+` before `container`

**Reply by lmilbaum** [reply]
Added. Same result.

**Reply by viceice** [reply]
please check debug logs, maybe the regex is still not matching.

**Reply by lmilbaum** [reply]
Thanks @viceice for your support

**Reply by lmilbaum** [reply]
The regex is matching according regex101. I'm unable to find effective information in the debug logs.

**Reply by lmilbaum** [reply, marked as answer]
Figured out the problem. The regex in the fileMatch didn't work.

**Reply by viceice** [reply]
🥳
